### DISCUSSION TITLE
Additions to the ECMAScript core for language-sensitive functionalities

### DISCUSSION THREAD

**Discussion by emphasis87** [sebastienros/jint#1445] [General]
Will the Intl global object be supported?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl

**Comment by lahma** [comment]
That is a ton of work.  Would you be interested to help out here?

**Reply by emphasis87** [reply]
That depends, if there will be someone to discuss things with, then perhaps I could spare some time. I am not familiar with Jint, so quick introduction where to put what would be welcome.

**Reply by lahma** [reply]
There's `intl`-branch I once made that shows most of the main constructs being introduced, without proper implementation:

https://github.com/sebastienros/jint/compare/main...lahma:jint:intl?expand=1

From there it would be going through everything reading [the ECMA-402 spec](https://tc39.es/ecma402/) and implementing features. There's some 2000+ failing tests covering the functionality.

**Reply by lahma** [reply]
Small update on this, I've merged the templates to main so it should be possible to implement parts needed as separate PRs now more easier.

**Reply by emphasis87** [reply]
Thank you, I was previously unclear on how to access the branch. If I find the time I will look into it, but my schedule is full for some weeks.

**Reply by Looooong** [reply]
Is it possible to use [Intl polyfills](https://formatjs.io/docs/polyfills/) instead?
